diff --git a/node_modules/react-native-torch/android/build.gradle b/node_modules/react-native-torch/android/build.gradle
index 3b19b4c..64dbf5f 100644
--- a/node_modules/react-native-torch/android/build.gradle
+++ b/node_modules/react-native-torch/android/build.gradle
@@ -16,5 +16,5 @@ android {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
 }
